Unix-linux-sysadmin-handbook-guide.md

```
@@ -0,0 +1,1560 @@
+# UNIX and Linux System Administration Handbook 実践ガイド
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)